ui/lib/kv/addon/components/kv-patch/json-form.hbs

```diff
@@ -12,7 +12,7 @@
       @message="JSON is unparsable. Fix linting errors to avoid data discrepancies."
     />
   {{/if}}
-  <hr class="has-background-gray-200" />
+
```

Comment:
Designs only had a horizontal line above buttons, not below JSON editor
![Screenshot 2024-08-26 at 3 14 11 PM](https://github.com/user-
attachments/assets/5d92485c-3ccf-4684-9303-dda40b48a9eb)